### DISCUSSION TITLE
:LazyHealth command ceasing to exist randomly.

### DISCUSSION THREAD

**Discussion by m-j-3** [LazyVim/LazyVim#2555] [Q&A]
![image](https://github.com/LazyVim/LazyVim/assets/114370635/20acdd89-421c-4646-95dc-443562b77cf4)

I've tried uninstalling and reinstalling both neovim and lazyvim a ton and :LazyHealth always works right off the bat but then within like the next 30-60mins it just starts giving the not an editor command message and i can't figure out what is causing it at all, would anyone happen to have any insight? 😓 I thought about making an issue for this but I have no idea how to write up a minimam reproduction for it since i cant seem to find any sort of specific trigger beyond it consistently occuring over time.

**Comment by dpetka2001** [comment]
Are you running the latest HEAD of LazyVim? Because it's not available in current stable release yet. It's scheduled for the next one.

**Reply by m-j-3** [reply]
I think this was it thank you 🙏
